### PR TITLE
Add Onboarding command

### PR DESCRIPTION
**Pull request by RecursiveHook**

### CODE REVIEW

File: cogs/autorole.py [hunk 1]
```diff
@@ -278,6 +281,13 @@ async def assign_roles_to_user(self,
                     # Send message after doing the above
                     # TODO: Ensure guarantee of removal before sending below...?
                     await ctx.send(f'{ctx.message.author.mention}', embed=role_embed)
+
+                    if success_callbacks:
+                        for cb in success_callbacks:
+                            try:
+                                await cb(user_matches[0])
+                            except Exception as e:
+                                logger.info(f'Success Callback Exception: {e}')
```
Comment (OxyOCE) [inline cogs/autorole.py:290]: Noice, it's a good pattern for now

File: cogs/autorole.py [hunk 2]
```diff
@@ -512,6 +522,71 @@ async def set_dj(self, ctx):
...
+        Can only be used by Vanguard (atm).
+        """
+
+        welcome_message = """Welcome to Ghost Proxy! <:ghost_proxy_2:455130686290919427>
+
+If you haven't yet, please read through our `#rules-conduct` and `#server-info` and then feel free to explore the server! A good place to start is `#voluspa`, our very own Warmind where you can set game roles, check who's online, and use a number of other helpful commands (type `$help`).
+
+If you have any questions about anything, feel free to ask in general chat, use the `$feedback` Voluspa feature (here in a DM even), or use the `@ghost-proxy-vanguard` ping to ask for help from the Ghost Proxy admin team.
+
+\\*\\*\\*
```
Comment (OxyOCE) [inline cogs/autorole.py:544]: Yo dawg... hehe

File: cogs/autorole.py [hunk 3]
```diff
@@ -512,6 +522,71 @@ async def set_dj(self, ctx):
...
+_ _
+        """
+
+        async def send_welcome_direct_message(user_rec):
+            new_member = self.bot.get_user(user_rec['id'])
+            welcome_prefix = f"_ _\n_ _\n" \
+                             f"Hello, {new_member.mention}! :wave: "
+            await new_member.send(f"{welcome_prefix}{welcome_message}")
+
+        async def send_welcome_guild_message(user_rec):
```
Comment (OxyOCE) [inline cogs/autorole.py:566]: Callback funcs look solid

File: cogs/autorole.py [hunk 4]
```diff
@@ -512,6 +522,71 @@ async def set_dj(self, ctx):
...
+            await guild_channel.send(guild_welcome)
+
+        await self.assign_roles_to_user(
+            ctx,
+            'ghost_proxy_roles',
+            [
+                'ghost-proxy-member'
+            ],
+            users,
+            role_limits=[
```
Comment (OxyOCE) [inline cogs/autorole.py:579]: Nice, patching up a few holes we've run into in the past

File: cogs/autorole.py (hunk 2 above)
Comment (RecursiveHook) [reply cogs/autorole.py:544]: \\:+1:\\

**Comment by RecursiveHook** [comment]
Sweet, thanks Oxy, merging. 🚢

**Pull request merged**